beta/src/pages/learn/writing-markup-with-jsx.md

```diff
@@ -64,58 +64,56 @@ export default function TodoList() {
 }
 ```
 
-If you copy and paste it as is, it will not work:
-
+यदि आप इसे वैसे ही कॉपी और पेस्ट करते हैं, तो यह काम नहीं करेगा:
 
 <Sandpack>
 
 ```js
 export default function TodoList() {
   return (
-    // This doesn't quite work!
-    <h1>Hedy Lamarr's Todos</h1>
-    <img 
-      src="https://i.imgur.com/yXOvdOSs.jpg" 
-      alt="Hedy Lamarr" 
+    // यह काम नहीं करता!
+    <h1>हेडी लैमर का टोडोस</h1>
+    <img
+      src="https://i.imgur.com/yXOvdOSs.jpg"
+      alt="हेडी लैमर"
       class="photo"
     >
     <ul>
-      <li>Invent new traffic lights
-      <li>Rehearse a movie scene
-      <li>Improve the spectrum technology
+      <li>नई ट्रैफिक लाइट का आविष्कार करें
+      <li>एक फिल्म के दृश्य का पूर्वाभ्यास करें
+      <li>स्पेक्ट्रम तकनीक में सुधार
     </ul>
   );
 }
 ```
 
 ```css
-img { height: 90px }
+img { height: 90px; }
 ```
 
 </Sandpack>
 
-This is because JSX is stricter and has a few more rules than HTML! If you read the error messages above, they'll guide you to fix the markup, or you can follow the guide below.
+ऐसा इसलिए है क्योंकि JSX सख्त है और इसमें HTML की तुलना में कुछ और नियम हैं! यदि आप ऊपर दिए गए त्रुटि संदेशों को पढ़ते हैं, तो वे मार्कअप को ठीक करने के लिए आपका मार्गदर्शन करेंगे, या आप नीचे दी गई मार्गदर्शिका का अनुसरण कर सकते हैं।
 
 <Note>
 
-Most of the times, React's on-screen error messages will help you find where the problem is. Give them a read if you get stuck!
-
+अधिकांश समय, React के ऑन-स्क्रीन त्रुटि संदेश आपको यह पता लगाने में मदद करेंगे कि समस्या कहाँ है। अगर आप फंस गए हैं तो उन्हें पढ़ लें!
 </Note>
 
-## The Rules of JSX {/*the-rules-of-jsx*/}
+## JSX के नियम {/_the-rules-of-jsx_/}
 
-### 1. Return a single root element {/*1-return-a-single-root-element*/}
+### 1. एकल मूल तत्व लौटाएं {/_1-return-a-single-root-element_/}
```

Comment:
`{/_1-return-a-single-root-element_/}` => `{/*1-return-a-single-root-element*/}`

lets not change this